badge-maker/README.md

```diff
@@ -1,42 +1,41 @@
-# gh-badges
+# badge-maker
 
-[![npm version](https://img.shields.io/npm/v/gh-badges.svg)](https://npmjs.org/package/gh-badges)
-[![npm license](https://img.shields.io/npm/l/gh-badges.svg)](https://npmjs.org/package/gh-badges)
+[![npm version](https://img.shields.io/npm/v/badge-maker.svg)](https://npmjs.org/package/badge-maker)
+[![npm license](https://img.shields.io/npm/l/badge-maker.svg)](https://npmjs.org/package/badge-maker)
 
 ## Installation
 
 ```sh
-npm install gh-badges
+npm install badge-maker
 ```
 
 ## Usage
 
 ### On the console
 
 ```sh
-npm install -g gh-badges
-badge build passed :green .png > mybadge.png
+npm install -g badge-maker
+badge build passed :green .svg > mybadge.svg
 ```
 
 ### As a library
 
 ```js
-const { BadgeFactory } = require('gh-badges')
-
-const bf = new BadgeFactory()
+const { makeBadge, ValidationError } = require('badge-maker')
 
 const format = {
   text: ['build', 'passed'],
   color: 'green',
   template: 'flat',
 }
 
-const svg = bf.create(format)
+makeBadge(format) // <svg...
+makeBadge({}) // ValidationError: Field `text` is required
```

Comment:
not a big deal, but may be helpful to include the assignment here at least on the initial use case like in the changelog and 2.x docs
```suggestion
const badge = makeBadge(format) // <svg...
makeBadge({}) // ValidationError: Field `text` is required
```